gammapy/data/event_list.py

```diff
@@ -93,6 +93,23 @@ def stack(cls, event_lists, **kwargs):
         stacked_table = vstack_tables(tables, **kwargs)
         return cls(stacked_table)
 
+    def stack(self, other):
+        """Stack with another EventList.
+
+        Calls `~astropy.table.vstack`.
+
+        Parameters
+        ----------
+        other : `~gammapy.data.EventList`
+            Event list to stack to self
+
+        Returns
+        -------
+        new_event_list : `~gammapy.data.EventList`
+            New event list
+        """
+        return self.__class__(vstack_tables([self.table, other.table]))
```

Comment:
On detail here: for the other classes we have such as `Map` or `Datasets`  the `.stack()` method work "in-place", this means it does not return anything but just modifies the existing data member. In this case you could just overwrite `self.table` and not return anything...

Reply:
done. Initially I did it not in place because that is the way it is in `GTI.stack()`...

Reply:
Good to know! I'll check once again, but likely we are then going to change it for `GTI` as well...

Reply:
I can do that too, if you want

Reply:
Yes, please go ahead. Thanks!